webapp/src/utils/styles.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
```

Comment:
Convert file to `.ts`. @avas27JTG Can you help Raghav in doing this?

Reply:
discussed